tables/afr-za-g1.ctb

```diff
@@ -15,10 +15,11 @@
 # accurate nor complete. It hasn't been verified by the table
 # author yet. It is merely an attempt by the liblouis maintainers
 # to get some sensible initial values in place.
-# ------------
 #
-# Based on World Braille Usage 2013
```

Comment:
Is this not based on World Braille Usage anymore?

Reply:
> All of the South African Braille codes, Afrikaans is just the first to get done, now use the UEB tables to represent the accented letters. So yes World Braille Usage is now out of date on this. It is correct to remove the reference to it.

Reply:
But shouldn't this be made clear in the comment instead of just removing the line?